doc/_static/diagrams/sphinx_build_flow.dot

```diff
@@ -13,26 +13,54 @@ digraph build {
         shape=record
         label = "Sphinx | <init> __init__ | <build> build"
     ];
+    "legend" [
+        shape=record
+        label = <<table border="0" cellborder="0" cellspacing="0">
+            <tr><td align="center"><u><b>Method types</b></u></td></tr>
+            <tr><td align="left"><font color="darkorange">Final</font></td></tr>
+            <tr><td align="left"><font color="darkblue">Overridable</font></td></tr>
+            <tr><td align="left"><font color="darkgreen">Abstract</font></td></tr>
+        </table>>
+    ];
+    {rank=same; "Sphinx" "legend" };
+
+    "Builder.init" [color=darkblue];
+    "Builder.build_all" [color=darkorange];
+    "Builder.build_specific" [color=darkorange];
+    "Builder.build_update" [color=darkorange];
```

Comment:
Is there a way to have an "alias" for the colors in general? this would perhaps in maintaining the file.

Reply:
no unfortunately not in "vanilla" graphviz, you would need to have a pre-build step like jinja or something

Reply:
I see :( well thanks for doing this! I only have little knowledge in graphviz so I appreciate you doing this.